Jenkinsfile

```diff
@@ -14,6 +14,12 @@ VAGOV_BUILDTYPES = [
   'vagovprod'
 ]
 
+DRUPAL_MAPPING = [
```

Comment:
I'm not sure if this is quite settled yet, but easy enough to update.. I would need to test, but if staging drupal ended up mapped to our dev and staging, we could just pass in a list in this dictionary.

Reply:
I think this is what we want.